examples/data_preprocessing/mimic_dataset/prepare_mimic_cxr.ipynb

```diff
@@ -0,0 +1,1520 @@
+    "# Preprocessing of MIMIC-CXR dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook illustrates how week supervision can be applied on X-rays and radiology reports.  \n",
+    "[MIMIC-CXR](https://physionet.org/content/mimic-cxr/2.0.0/) Database is a large publicly available dataset of chest X-rays including radiology reports. It contains 377110 images and 227835 radiographic studies.\n",
+    "[MIMIC-CXR-JPG](https://physionet.org/content/mimic-cxr-jpg/2.0.0/) Database also includes weak labels which are derived from the radiology reports using CheXpert labler. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting all set up"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get access to the data, one needs to be a \"credential user\" in PhysioNet and sign the data use agreement. \n",
+    "1. To get a credentialed PhysioNet account, follow the following [instructions](https://physionet.org/about/citi-course/). \n",
+    "2. Then sign the data use agreement for both, the MIMIC-CXR and the MIMIC-CXR-JPG database. \n",
+    "3. Download the files\n",
+    "    \"cxr-record-list.csv.gz\" (from mimic-cxr),\n",
+    "    \"cxr-study-list.csv.gz\" (from mimic-cxr),\n",
+    "    \"mimic-cxr-2.0.0-split.csv.gz\" (from mimic-cxr-jpg),\n",
+    "    \"mimic-cxr-2.0.0-chexpert.csv.gz\" (from mimic-cxr-jpg)\n",
+    "    and unzip them with 7zip   \n",
+    "4. set your working directory to this folder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#os.chdir(\"\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import random\n",
+    "import copy\n",
+    "import torch.optim as optim\n",
+    "import csv\n",
+    "import torchvision.transforms as transforms\n",
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torchvision.models as models\n",
+    "import itertools\n",
+    "from tqdm import tqdm\n",
+    "from PIL import Image\n",
+    "from torch.utils.data import DataLoader, Dataset\n",
+    "from typing import Dict\n",
+    "from joblib import dump, load"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Downloading MIMIC-CXR-JPG images"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To download the images with the following code, a credentialed PhysioNet account is required. Before executing the following you should do the following:\n",
+    "1. check if you have enough memory space in the current environment (the first 1000 images are 1.51GB large)\n",
```

Comment:
I meant the current directory, since the images will be downoaded there. I changed it to "check if you have enough memory space in the current directory"